qemu/deps/cpuid/cpuid_dump.txt

```diff
@@ -1,20 +1,20 @@
 x86       Opteron_G5 AMD Opteron 63xx class CPU
   family 21 model 2 stepping 0 level 13 xlevel 0x80000001a vendor "AuthenticAMD"
-  feature_edx (sse2 sse fxsr mmx clflush pse36 pat cmov mca pge mtrr sep apic cx8 mce pae msr tsc pse de fpu)
-  feature_ecx (f16c avx xsave aes popcnt sse4.2|sse4_2 sse4.1|sse4_1 cx16 pma ssse3 pclmulqdq|pclmuldq pni|sse3)
+  feature_edx (vme sse2 sse fxsr mmx clflush pse36 pat cmov mca pge mtrr sep apic cx8 mce pae msr tsc pse de fpu)
```

Comment:
some texts from cpu.c in blew: 
.name = "Opteron_G5",
        ......       
       .features[FEAT_1_EDX] =
            CPUID_VME | CPUID_SSE2 | CPUID_SSE | CPUID_FXSR | CPUID_MMX |
            .......